web/.browserslistrc

```diff
@@ -1,3 +1,2 @@
 > 1%
 last 2 versions
-not dead
```

Comment:
This change solves [this issue](https://qiita.com/kubotama/items/3af6c4d08ba83ef21c30#%E3%83%93%E3%83%AB%E3%83%89%E3%81%AE%E3%82%A8%E3%83%A9%E3%83%BC%E3%82%92%E8%A7%A3%E6%B6%88%E3%81%99%E3%82%8B)